.github/workflows/ci.yml

```
@@ -0,0 +1,22 @@
+name: CI
+on: [pull_request]
+
+jobs:
+    buildinfo:
+        runs-on: ubuntu-latest
+        container: debian:bullseye
+        steps:
+            - name: Install dependencies
+              run: |
+                apt-get update && apt-get install python3-debian git git-lfs -y
```

Nit: I find (long) arguments first to be more readable scripted invocations, but your style may vary: :-)

```suggestion
                apt-get update && apt-get install --yes python3-debian git git-lfs
```